spec/mongoid/association/embedded/embedded_in/proxy_spec.rb

```diff
@@ -616,4 +616,31 @@
       end
     end
   end
+
+  context "when assigning a hash" do
```

Comment:
Suggest to add (1) aliased and (2) localized fields to all specs.

(I know it's a pain but they always tend to break :)